content/en/blog/_posts/2023-07-18-swap-beta1-graduation/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: blog
+title: "Kubernetes 1.28: NodeSwap graduates to Beta"
+date: 2023-07-18
```

Comment:
The date and title should be changed according to release team's comment later.

I still think that the title `NodeSwap graduates to Beta` is not that accuracy. 
- But beta1 is also not clear.
- This is beta but disabled by default.

Not sure if there is a better suggestion here.

Reply:
/hold

pending assignment of a blog article publication date, and this being updated (including the filename) to match.